js/renovation/check-box.tsx

```diff
@@ -0,0 +1,188 @@
+import {
+  Component,
+  ComponentBindings,
+  JSXComponent,
+  Method,
+  OneWay,
+  TwoWay,
+  Ref,
+  Effect,
+} from 'devextreme-generator/component_declaration/common';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { h } from 'preact';
```

Comment:
Don't forget to remove this import after a merge with 20_2
It unnecessary after this commit
https://github.com/DevExpress/DevExtreme/commit/6a9f68a0ee92b6b62b81194e7198e0460132aa5c

Reply:
Thanks!

Reply:
fixed